Exercise_A/index.html

```diff
@@ -1,16 +1,23 @@
 <!DOCTYPE html>
 <html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="" />
-  </head>
-  <body>
-    <!-- CODE HERE -->
 
-    <script src="" async defer></script>
-  </body>
-</html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title></title>
+  <meta name="description" content="" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="" />
+</head>
+
+<body>
+  <!-- CODE HERE -->
+  <a href="about.html">
+    <img
+      src="https://img.icons8.com/external-itim2101-blue-itim2101/452/external-contact-us-contact-and-message-itim2101-blue-itim2101-4.png"
+      alt="about">
+  </a>
+  <script src="" async defer></script>
+</body>
+
+</html>
```

Comment:
You did a good job adding an anchor tag and linking it to another HTML page. Well done!